File: Examples/test-suite/ruby/grouping_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# Author::    gga
# Copyright:: 2007
# License::   SWIG
#

require 'swig_assert'

require 'grouping'

x = Grouping.test1(42)
if x != 42
  raise RuntimeError
end

Grouping.test2(42)

x = Grouping.do_unary(37, Grouping::NEGATE)
if x != -37
  raise RuntimeError
end

Grouping.test3 = 42

File: Examples/test-suite/javascript/grouping_runme.js
var grouping = require("grouping");

var x = /* await */(grouping.test1(42));
if (x != 42)
    throw new Error;

/* await */(grouping.test2(42));

var x = /* await */(grouping.do_unary(37, grouping.NEGATE));
if (x != -37)
    throw new Error;

grouping.test3 = 42;
if (grouping.test3 !== 42)
    throw new Error;
